winml/api/windows.ai.machinelearning.native.internal.idl

```diff
@@ -59,6 +59,7 @@ interface ILearningModelEvaluationResultNative : IUnknown
 interface ILearningModelSessionNative : IUnknown
 {
     HRESULT GetIntraOpNumThreads(UINT32* numThreads);
+    HRESULT GetIntraOpThreadSpinning(boolean* allowSpinning);
```

Comment:
This should be ILearningModelSessionNative1

Reply:
fixed